src/natsp.h

```diff
@@ -699,6 +699,9 @@ struct __natsConnection
     natsHash            *subs;
     natsMutex           *subsMu;
 
+    microService        **services;
```

Comment:
0/5 These take up memory in all connections; would be nice to come up with a better mechanism of attaching extended data to connections, without the fixed overhead.